docs/pages/example/animate-point-along-route.html

```diff
@@ -115,7 +121,8 @@
         "type": "symbol",
         "layout": {
             "icon-image": "airport-15",
-            "icon-rotate": 90,
+            "icon-rotate": { "type": "identity", "property": "bearing"},
```

Comment:
Replace the function with `["get", "bearing"]`. (We decided to always use expressions in examples.)